.github/workflows/tests_linux_x86_mpich.yml

```diff
@@ -115,7 +115,7 @@ jobs:
           which mpiexec
           module unload mpich
 
```

Comment:
Why do we need `Validate Multi-GPU packages` in `./tests_linux_x86.yml`? And why discover `openmpi` in the MPICH workflow and vice versa?

Reply:
It's an implementation in PR#118 and it was not changed since it's not focus of this PR.